packages/adapter-neon/src/index.ts

```diff
@@ -0,0 +1,261 @@
+/**
+ * <div style={{display: "flex", justifyContent: "space-between", alignItems: "center", padding: 16}}>
+ *  <p>An official <a href="https://www.postgresql.org/">PostgreSQL</a> adapter for Auth.js / NextAuth.js.</p>
+ *  <a href="https://www.postgresql.org/">
+ *   <img style={{display: "block"}} src="/img/adapters/neon.svg" width="48" />
+ *  </a>
+ * </div>
+ *
+ * ## Installation
+ *
+ * ```bash npm2yarn
+ * npm install next-auth @auth/neon-adapter
+ * ```
+ *
+ * @module @auth/neon-adapter
+ */
+
+import type {
+  Adapter,
+  AdapterUser,
+  VerificationToken,
+  AdapterSession,
+} from "@auth/core/adapters"
+import type { Pool } from "@neondatabase/serverless"
+
+export function mapExpiresAt(account: any): any {
+  const expires_at: number = parseInt(account.expires_at)
+  return {
+    ...account,
+    expires_at,
+  }
+}
```

Comment:
You can import `AdapterAccount` type from the `@auth/core/adapters` as well. And then `account.expires_at` is `number | undefined` so `parseInt` shouldn't be required :pray: 

```suggestion
export function mapExpiresAt(account: AdapterAccount): AdapterAccount {
  const expires_at = account.expires_at
  return {
    ...account,
    expires_at,
  }
}
```